lib/gitarro/git_op.rb

```diff
@@ -47,9 +47,14 @@ def del_pr_branch(upstream, pr)
   private
 
   def ck_or_clone_git
-    return if File.directory?(git_dir)
-    FileUtils.mkdir_p(git_dir)
+    git_repo_dir = git_dir + '/' + @options[:repo].split('/')[1]
+    return if File.directory?(git_repo_dir)
+    FileUtils.mkdir_p(git_dir) if File.directory?(git_dir)
     Dir.chdir git_dir
+    clone_repo
+  end
+
+  def clone_repo
     repo_url = "#{repo_protocol}#{@options[:repo]}.git"
     puts `git clone #{repo_url}`
```

Comment:
just a small nitpick ( since we are already here in the code base.)

I think we could add a check that if the git clone repo command mess up, we exit with 1 and some logging.

Look at english library

Reply:
Mmmm... now you mention it, I am pretty sure we already add it before. Didn't we?

Reply:
something like this https://github.com/meaksh/gitbot/blob/50cdcefc33a5974982427be05e3dcd1f7cdc9f2c/lib/gitarro/git_op.rb#L115

Reply:
Nah, it was at #50 which is a complete rewrite of git_opt.rb, but still WIP :-D

Reply:
yop , but better do small changes and this is not to intruisive.

Reply:
@juliogonzalez, @MalloZup - Added! Thanks for the feedback! 🍰

Reply:
@meaksh  please do not add :cake:  , :heart_eyes: